doc/data/messages/e/exec-used/bad.py

```diff
@@ -1,4 +1,5 @@
-username = "Ada"
-code_to_execute = f"""input('Enter code to be executed please, {username}: ')"""
-program = exec(code_to_execute)  # [exec-used]
-exec(program)  # [exec-used]
+def get_user_code(name):
+    return input(f"Enter code to be executed please, {name}: ")
+
+
+exec(get_user_code("Ada"))  # [exec-used]
```

Comment:
```suggestion
username = "Ada"
code_to_execute = f"""input('Enter code to be executed please, {username}: ')"""
program = exec(code_to_execute)  # [exec-used]
exec(program)  # [exec-used]
```
I see the logic now of removing a useless exec from the bad.py.